self_enrollment_tool/views.py

```diff
@@ -176,3 +256,20 @@ def enroll (request, course_instance_id):
         'abort': False,
     }
     return render(request, 'self_enrollment_tool/lookup.html', context)
+
+
+@login_required
+@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_XLIST_TOOL)
+@require_http_methods(['GET'])
+def remove_self_enroll(request, pk):
+    try:
+        self_enrollment_course = SelfEnrollmentCourse.objects.get(pk=pk)
+        self_enrollment_course.delete()
+    except Exception as e:
+        msg = f'Unable to remove self enrollment course {pk}.'
+        logger.exception(msg)
+        logger.exception(e)
+        messages.error(request, msg)
+
+    return redirect('self_enrollment_tool:index')
```

Comment:
Notes from our Zoom conversation:
- Rename view to be a little clearer
- Change permission for lti_permission_required to be PERMISSION_SELF_ENROLLMENT_TOOL
- Add exception for object DoesNotExist and return a warning message
- Add logging statement for deleting SelfEnrollmentCourse
- Remove logger.exception(e) as it is redundant to the exception log above it
- Make the HTTP method DELETE